docs/docs-content/vertex/system-management/add-registry.md

```diff
@@ -0,0 +1,63 @@
+---
+sidebar_label: "Add a Registry"
+title: "Add a Registry"
+description: "Learn how to add a registry in Palette VerteX."
+icon: ""
+hide_table_of_contents: false
+sidebar_position: 50
+tags: ["vertex", "management", "registry"]
+---
+
+You can add a registry at the system level or tenant level. Registries added at the system level are available to all the tenants. Registries added at the tenant level are available only to that tenant. This section describes how to add a system-level registry. For guidance, check out [Add a Tenant-Level Registry](../../tenant-settings/add-registry.md).
+
+## Prerequisites
+
+- You need tenant admin privileges. 
```

Comment:
```suggestion
-  Access to the Palette VerteX system console.
```
Only those with access to the system console can create a registry in the system console, not tenant admins.

Reply:
Added